source/fe/mapping_c1.cc

```diff
@@ -66,7 +66,7 @@ MappingC1<2>::MappingC1Generic::add_line_support_points (const Triangulation<2>:
                                                          std::vector<Point<2> > &a) const
 {
   const unsigned int dim = 2;
-  std::vector<Point<dim> > line_points (2);
+  static const QGaussLobatto<1> quad_points(4);
```

Comment:
Do we have to make this variable static? I do not like to bloat programs with unnecessary static variables. Since the meaning of GL at this point is rather arbitrary, you could just as well use the two points `t1=0.5-sqrt(1./20)` and `t2=0.5+sqrt(1./20)` explicitly, thus avoiding any constructions.